vue-frontend/src/components/home-new/BlogSection.vue

```diff
@@ -0,0 +1,222 @@
+              class="tw-hidden md:tw-block tw-absolute tw-right-10 md:tw-right-[0.5rem] xl:-tw-right-[1.5rem] 2xl:-tw-right-[5.5rem] tw-w-[20rem] lg:tw-w-[23rem] xl:tw-w-[31.875rem] tw-max-h-[13.75rem] xl:tw-max-h-[20rem] tw-animate-fadeIn"
+              loading="lazy"
+              :alt="blog.title"
+            />
+          </a>
+          <div class="tw-flex tw-flex-row tw-gap-8">
+            <div class="tw-flex tw-flex-col tw-pt-[1.5rem] md:tw-pt-[2.120rem]">
+              <div
+                class="tw-font-inter-semibold md:tw-font-inter-regular tw-text-[1.5rem] md:tw-text-[2.188rem] 2xl:tw-text-[3.125rem] tw-leading-[1.95rem] md:tw-leading-[2.183rem] 2xl:tw-leading-[4.063rem]"
+              >
+                {{ blog.pubDate[1] }}
+              </div>
+              <div
+                class="tw-text-center font-inter-medium md:tw-font-inter-regular tw-text-[0.625rem] md:tw-text-[0.875rem] 2xl:tw-text-[1.125rem] tw-leading-[0.8125rem] md:tw-leading-[1.125rem] 2xl:tw-leading-[1.463rem]"
+              >
+                {{ blog.pubDate[0] }}
+              </div>
+            </div>
+            <div class="tw-flex tw-flex-col">
+              <div
+                class="tw-mt-[0.9375rem] md:tw-mt-[1.813rem] tw-w-[16.5rem] sm:tw-w-[19.5rem] lg:tw-w-[27.5rem] 2xl:tw-w-[43.875rem] tw-font-inter-semibold tw-text-[1.125rem] md:tw-text-[1.688rem] 2xl:tw-text-[2.125rem] tw-leading-[1.575rem] md:tw-leading-[2.313rem] 2xl:tw-leading-[3.188rem]"
+              >
+                <span
+                  :class="
+                    activeIndex == index
+                      ? 'md:tw-bg-gradient-to-b md:tw-from-[#ff9472] md:tw-to-[#f2709c] md:tw-bg-[length:100%_5px] md:tw-bg-no-repeat md:tw-bg-bottom'
+                      : ''
+                  "
+                  class="tw-pb-[0.3rem] md:hover:tw-bg-gradient-to-b hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-bg-[length:100%_5px] hover:tw-bg-no-repeat hover:tw-bg-bottom"
+                >
+                  {{ blog.title }}</span
+                >
+              </div>
+              <div
+                class="tw-flex tw-flex-row tw-gap-[9rem] lg:tw-gap-72 2xl:tw-gap-[31rem]"
+              >
+                <div
+                  class="tw-mt-[0.9375rem] tw-w-[4.5rem] 2xl:tw-w-[5rem] tw-font-inter-medium tw-text-[0.625rem] md:tw-text-[0.875rem] 2xl:tw-text-[1.125rem] tw-leading-[0.8125rem] md:tw-leading-[1.125rem] 2xl:tw-leading-[1.463rem]"
+                >
+                  {{ blog.author }}
+                </div>
+                <div
+                  class="tw-mt-[0.9375rem] tw-w-[3.25rem] md:tw-w-[4.5rem] 2xl:tw-w-[5.938rem] tw-font-inter-medium tw-text-[0.625rem] md:tw-text-[0.875rem] 2xl:tw-text-[1.125rem] tw-leading-[0.8125rem] md:tw-leading-[1.125rem] 2xl:tw-leading-[1.463rem]"
+                >
+                  Read more
+                </div>
+              </div>
+            </div>
+          </div>
+        </a>
+        <hr
+          v-if="
```

Comment:
No need for it, as we have put one hr above.
Only 1 line is needed after all blogs. And for mobile devices, condition can be simplified like `if the index is the last index hide it`